example/example-model/src/main/java/uk/gov/gchq/palisade/example/client/ExampleSimpleClient.java

```diff
@@ -165,8 +169,11 @@ private void initialiseServices() {
         return super.read(filename, RESOURCE_TYPE, userId, justification);
     }
 
-    @Override
-    protected Serialiser<ExampleObj> createSerialiser() {
-        return new ExampleObjSerialiser();
+    private ParentResource getParent(final String fileURL) {
+        if (fileURL.isEmpty()) {
+            return new SystemResource().id("/");
```

Comment:
Can the root ID be changed to FileSystems.getDefault().getRootDirectories()... or similar?